wyvern/components/component.py

```diff
@@ -159,12 +173,20 @@ def get_feature(
             you just have to pass in feature_name="wyvern_feature".
         """
         current_request = request_context.ensure_current_request()
-        feature_data = current_request.feature_map.feature_map.get(identifier)
-        if not feature_data:
-            return None
-        return feature_data.features.get(feature_name)
+        feature_name = feature_name.replace(":", "__")
```

Comment:
I was doing it because I got an error while doing the initial implementation. Looking back, can't remember the error and couldn't find anything related online. I reverted those changes and it works with `:` atm.